cpp/src/feather/CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ set(LIBFEATHER_SRCS
   status.cc
   types.cc
   writer.cc
+  api.h
+  buffer.h
+  common.h
+  feather-c.h
+  io.h
+  metadata.h
+  reader.h
+  status.h
+  types.h
+  writer.h
```

Comment:
Hate to get nitpicky, but can you put the headers in a separate list (`set(LIBFEATHER_HEADERS ...`), then you can list `${LIBFEATHER_HEADERS}` after `${LIBFEATHER_SRCS}` which should do the trick. Thanks! =)